server/src/test/java/com/objectcomputing/checkins/services/memberskill/MemberSkillServiceImplTest.java

```diff
@@ -0,0 +1,272 @@
+package com.objectcomputing.checkins.services.memberskill;
+
+import com.objectcomputing.checkins.services.memberSkill.MemberSkill;
+import com.objectcomputing.checkins.services.memberSkill.MemberSkillRepository;
+import com.objectcomputing.checkins.services.memberSkill.MemberSkillServiceImpl;
+import com.objectcomputing.checkins.services.memberSkill.MemberSkillBadArgException;
+import com.objectcomputing.checkins.services.memberprofile.MemberProfile;
+import com.objectcomputing.checkins.services.memberprofile.MemberProfileRepository;
+import com.objectcomputing.checkins.services.skills.Skill;
+import com.objectcomputing.checkins.services.skills.SkillRepository;
+import io.micronaut.test.annotation.MicronautTest;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@MicronautTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class MemberSkillServiceImplTest {
```

Comment:
No test case for the "member already has this skill" case.

Reply:
👍